Esri/arcgis-maps-sdk-swift-samples
# [New] Identify raster cell

## Description

This PR implements `Identify raster cell` in `Layers` category.
URL to README: [README](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-IdentifyRasterCell/Shared/Samples/Identify%20raster%20cell)

## Linked Issue(s)

- `swift/issues/4596`

## How To Test

- Tap on the raster layer to see the popup for the raster cell at the tap location.
- Tap and hold to see map magnifier and the popup update dynamically as you drag.

## Screenshots

![identify-raster-cell](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/751aa9c4-3510-4900-b18b-7eb25ad4f3b6)
